MyOnboarding/test/MyOnboardingApp.Api.Tests/MyOnboardingApp.Api.Tests.csproj

```diff
@@ -66,16 +76,23 @@
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="TodoListControllerTest.cs" />
+    <Compile Include="Utils\ItemEqualityComparer.cs" />
     <Compile Include="Utils\TodoListControllerExtensions.cs" />
```

Comment:
Is this generic extension really ToDoListController-specific? :stuck_out_tongue: